ResourcesCard.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect } from "react";
+
+import { sendEvt } from "services/event.service";
+
+import imgFallback from "img/resource_fallback.png";
+
+import styles from "./ResourcesCard.module.scss";
+
+const ResourcesCard = () => {
+  const [resources, setResources] = useState([]);
+  const [primaryResource, setPrimaryResource] = useState(none);
+  const [secondaryResources, setSecondaryResources] = useState([]);
+
+  var sendEventTracking = (resourceUuid) => {
+    const evt = new Object();
+    evt = {
+      event_name: "web_dashboard_card_" + r.id + "_" + r.title + "_tap"
```

Comment:
We can leverage [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to make this more readable:

```suggestion
      event_name: `web_dashboard_card_${r.id}_${r.title}_tap`
```